dev/DynamicDependencyLifetimeManager/DynamicDependencyLifetimeManager/DynamicDependencyLifetimeManager.rc

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#include <winresrc.h>
+#define VERSIONINFO_FILENAME "DynamicDependencyLifetimeManager.exe"
+#define VERSIONINFO_FILETYPE VFT_APP
+#include "..\..\..\eng\common\VersionInfo\AssemblyInfo.ver"
```

Comment:
Why "AssemblyInfo"? That's a very .NET term

SUGGESTION: `ProductVersionInfo.rc`

Reply:
You mean ProductVersionInfo.ver?

I think it's too late for this.

Reply:
It's never too late... ;-)

Maybe high cost / not worth it near term. Still, as a C++ developer "assembly" is an odd turn of phrase :-( But...maybe another day

Reply:
I shall do it on another PR since the change needs to be made on the aggregator repo